components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -11604,4 +11604,47 @@ public static void initializeVelocityContext(VelocityEngine velocityEngine){
         velocityEngine.setProperty("runtime.conversion.handler", "none");
 
     }
+
+    /**
+     * Handles gateway vendor for Choreo Connect before insert DB operations.
+     *
+     * @param gatewayVendorType Gateway vendor
+     * @param gatewayType       Gateway type
+     * @return Choreo Connect gateway vendor
```

Comment:
This returns not only for the choreo connect vendor, but for all the vendors. Please use a gateway neutral wording.